Tasks/ApiCompat/readme.md

```diff
@@ -72,6 +72,11 @@ For the build trigger configure  **Pull request validation**.
 
 With this configuration, whenever a person opens a Pull Request introducing changes that will break compatibilities with the current released version, the build process will be marked as ***Failed***.
 
+#### Unexpected behavior
```

Comment:
I think we should have this title at the same level as `Basic Example` or `More Information`
```suggestion
## Unexpected Behavior
```